nbconvert/utils/pandoc.py

```diff
@@ -109,6 +109,28 @@ def check_pandoc_version():
                        RuntimeWarning, stacklevel=2)
     return ok
 
+try: 
+    from pandocfilters import applyJSONFilters
+except ImportError:
```

Comment:
John did a new release of pandocfilters for us, so we can rely on `>=1.4.1` and drop the falllback here.